pkg/route/hhg_planner.go

```
@@ -63,6 +63,16 @@ func (p *hhgPlanner) ZipTransitDistance(appCtx appcontext.AppContext, source str
 		return p.dtodPlannerMileage.DTODZip5Distance(appCtx, source, destination)
 	}
 
+	// Get reZip3s for origin and destination to compair base point cities.
+	// Dont worry about errors, if we dont find them, we'll just use randMcNallyZip3Distance
+	sourceReZip3, _ := models.FetchReZip3Item(appCtx.DB(), sourceZip3)
+	destinationReZip3, _ := models.FetchReZip3Item(appCtx.DB(), destZip3)
```

Might be worth logging errors other than sqlNoRows here, even if the function can go on, to avoid hiding issues with the database. I don't have strong feelings about this though.

---

Good call, I agree that it would be best to at least log out errors so they dont get totally swallowed, even if we can continue on after the error. Fixed!